testsuite/MDAnalysisTests/topology/test_parmed.py

```diff
@@ -168,16 +177,22 @@ def test_dihedrals_identity(self, top, value):
     def test_cmaps_identity(self, top, value):
         vals = top.cmaps.values
         assert value in vals or value[::-1] in vals
-    
+
+
 class TestParmedParserPRM(BaseTestParmedParser):
     """
     PRM
     """
-    
+
     ref_filename = PRM
-    
+
     expected_n_atoms = 252
     expected_n_residues = 14
+    elems_ranges = ((0, 8), (30, 38))
```

Comment:
So strangely enough Parmed here actually guesses elements from masses (hence why PRM directly loaded in MDA has no elements, but this yields an element list). But since it's outside our hands... I guess it's fine?

Reply:
I agree, IMO the goal should be to faithfully represent the ParmEd object, however it chooses to interpret the original file.